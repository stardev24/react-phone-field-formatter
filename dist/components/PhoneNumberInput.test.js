import React from "react";
import PhoneTextInput from "./PhoneNumberInput";
import renderer from "react-test-renderer";
describe("PhoneTextInput", function () {
  it("renders properly", function () {
    var tree = renderer.create(React.createElement(PhoneTextInput, {
      label: "Phone",
      placeholder: "(541) 754-3010"
    })).toJSON();
    expect(tree).toMatchSnapshot();
  });
});