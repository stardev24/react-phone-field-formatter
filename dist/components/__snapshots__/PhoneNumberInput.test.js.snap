// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoneTextInput renders properly 1`] = `
Array [
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline simple-text-input"
    onClick={[Function]}
  >
    <input
      className="MuiInputBase-input MuiInput-input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="(541) 754-3010"
      type="text"
      value=""
    />
  </div>,
  <p
    id="notify"
  >
    
  </p>,
]
`;
